topics/templates/document-attributes-product.adoc

```diff
@@ -2,8 +2,8 @@
 :project_community: false
 :project_product: true
 :project_version: 7.2.0.GA
-:project_versionMvn: 3.4.2.Final-redhat1
-:project_versionNpm: 3.4.2.Final-redhat1
+:project_versionMvn: 3.4.2.Final-redhat-2
```

Comment:
Hey @matthewhelmke, the version should be 3.4.3.Final-redhat-2, not 3.4.2.Final-redhat-2

Reply:
Thank you.